### PR TITLE
add createCharacterCard function

### PR DESCRIPTION
**Pull request by ninaassmann**
add createCharacterCard function

### CODE REVIEW

**Comment by jonas8900** [comment]
this looks amazing !

File: components/card/card.js [hunk 1]
```diff
@@ -1 +1,35 @@
-export function createCharacterCard() {}
+const cardContainer = document.querySelector('[data-js="card-container"]');
+
+export function createCharacterCard(
+  charName,
+  source,
+  status,
+  type,
+  occurrences
+) {
```
Comment (jonas8900) [inline components/card/card.js:9]: hmm.. this could be prettier......

**Comment by KilianBau** [comment]
LGTM!

**Comment by chris-kuehn** [comment]
LGTM! :)

**Pull request merged**